fix missing redirect to create cdr1as_test.bam in test_data subdirectory
see also https://github.com/marvin-jens/find_circ/pull/13
author: Kenneth Hoste (HPC-UGent)
--- test_data/Makefile.orig	2017-11-16 18:50:23.433979000 +0100
+++ test_data/Makefile	2017-11-16 18:50:31.876523000 +0100
@@ -21,7 +21,7 @@
 	@printf "\n>>> aligning example reads\n\n"
 	bowtie2 -p8 --very-sensitive --score-min=C,-15,0 --reorder --mm \
 		-f -U cdr1as_reads.fa -x bt2_cdr1as_locus \
-		2> bt2_firstpass.log | samtools view -hbuS - | samtools sort - cdr1as_test
+		2> bt2_firstpass.log | samtools view -hbuS - | samtools sort - > cdr1as_test.bam
 
 unmapped_cdr1as_test.bam: cdr1as_test.bam
 	@printf "\n>>> fetching the unmapped reads\n\n"
